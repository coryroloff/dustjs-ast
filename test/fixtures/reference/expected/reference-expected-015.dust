{[1].thing[1]}