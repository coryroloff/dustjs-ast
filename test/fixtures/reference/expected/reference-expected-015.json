{
	"type": "Template",
	"body": [
		{
			"type": "Reference",
			"id": {
				"type": "Identifier",
				"path": [
					{
						"type": "Key",
						"name": "one",
						"loc": {
							"start": {
								"line": 1,
								"column": 8
							},
							"end": {
								"line": 1,
								"column": 11
							}
						}
					},
					{
						"type": "Key",
						"name": "two",
						"loc": {
							"start": {
								"line": 1,
								"column": 12
							},
							"end": {
								"line": 1,
								"column": 15
							}
						}
					},
					{
						"type": "Key",
						"name": "three",
						"loc": {
							"start": {
								"line": 1,
								"column": 16
							},
							"end": {
								"line": 1,
								"column": 21
							}
						}
					},
					{
						"type": "Key",
						"name": "four",
						"loc": {
							"start": {
								"line": 1,
								"column": 22
							},
							"end": {
								"line": 1,
								"column": 26
							}
						}
					},
					{
						"type": "Key",
						"name": "five",
						"loc": {
							"start": {
								"line": 1,
								"column": 27
							},
							"end": {
								"line": 1,
								"column": 31
							}
						}
					}
				],
				"localContext": false,
				"loc": {
					"start": {
						"line": 1,
						"column": 8
					},
					"end": {
						"line": 1,
						"column": 31
					}
				}
			},
			"filters": [],
			"loc": {
				"start": {
					"line": 1,
					"column": 1
				},
				"end": {
					"line": 1,
					"column": 35
				}
			}
		}
	],
	"tokens": [
		{
			"type": "Punctuator",
			"value": "{",
			"loc": {
				"start": {
					"line": 1,
					"column": 1
				},
				"end": {
					"line": 1,
					"column": 2
				}
			}
		},
		{
			"type": "Key",
			"value": "one",
			"loc": {
				"start": {
					"line": 1,
					"column": 8
				},
				"end": {
					"line": 1,
					"column": 11
				}
			}
		},
		{
			"type": "Punctuator",
			"value": ".",
			"loc": {
				"start": {
					"line": 1,
					"column": 11
				},
				"end": {
					"line": 1,
					"column": 12
				}
			}
		},
		{
			"type": "Key",
			"value": "two",
			"loc": {
				"start": {
					"line": 1,
					"column": 12
				},
				"end": {
					"line": 1,
					"column": 15
				}
			}
		},
		{
			"type": "Punctuator",
			"value": ".",
			"loc": {
				"start": {
					"line": 1,
					"column": 15
				},
				"end": {
					"line": 1,
					"column": 16
				}
			}
		},
		{
			"type": "Key",
			"value": "three",
			"loc": {
				"start": {
					"line": 1,
					"column": 16
				},
				"end": {
					"line": 1,
					"column": 21
				}
			}
		},
		{
			"type": "Punctuator",
			"value": ".",
			"loc": {
				"start": {
					"line": 1,
					"column": 21
				},
				"end": {
					"line": 1,
					"column": 22
				}
			}
		},
		{
			"type": "Key",
			"value": "four",
			"loc": {
				"start": {
					"line": 1,
					"column": 22
				},
				"end": {
					"line": 1,
					"column": 26
				}
			}
		},
		{
			"type": "Punctuator",
			"value": ".",
			"loc": {
				"start": {
					"line": 1,
					"column": 26
				},
				"end": {
					"line": 1,
					"column": 27
				}
			}
		},
		{
			"type": "Key",
			"value": "five",
			"loc": {
				"start": {
					"line": 1,
					"column": 27
				},
				"end": {
					"line": 1,
					"column": 31
				}
			}
		},
		{
			"type": "Punctuator",
			"value": "}",
			"loc": {
				"start": {
					"line": 1,
					"column": 34
				},
				"end": {
					"line": 1,
					"column": 35
				}
			}
		}
	],
	"loc": {
		"start": {
			"line": 1,
			"column": 1
		},
		"end": {
			"line": 1,
			"column": 35
		}
	}
}